alerter.js

```diff
@@ -120,35 +133,86 @@
             conf = { 'text' : conf };
         }
 
-        activeAlerts += 1;
+        ++activeAlerts;
 
         options = extend(defaults, conf);
 
         container = document.createElement('div');
 
-        if(options.orientation === 'left') {
+        if (options.id)   {
+            if (typeof options.id === 'string') {
+                container.id = options.id;
+            }
+            else {
+                return console.error('Bad id.');
+            }
+        }
+
+        if (options.cssClassName)   {
```

Comment:
No, it can't be because ```typeof '' == "string"```, so you still have to check if it's true or not or if it's different than ''/"".
Nested conditions are not necessary, I'll give you that, I'm sometimes too careful due to my low-level experience in C/C++.

Reply:
Ignore it on invalid classname is a good solution. I'll do that.